### DISCUSSION TITLE
Can't execute an application

### DISCUSSION THREAD

**Discussion by omar909** [i3/i3#5994] [Q&A]
I tried adding to i3config file 
`bindsym $mod+g exec firefox` 
But that didn't work what do I do

**Comment by levkush** [comment]
Try adding `--no-startup-id` tag to the command like this: `bindsym $mod+g exec --no-startup-id firefox`

Mark as completed if it solved your question

**Reply by okraits** [reply]
1. Does Firefox start if you execute `firefox` in a terminal?
2. Does the keybinding work if you use `exec i3-sensible-terminal` instead of `exec firefox`?
3. Did you reload or restart i3 after editing your configuration file?
